CONTRIBUTING.md

```diff
@@ -1,13 +1,37 @@
 ## How to Contribute to Scene Builder
 
-Contribution of any form is welcome! Please see the list below on how you can contribute to the project. Once you've decided what you would like to do, let us know about it first in the [Discussions section](https://github.com/gluonhq/scenebuilder/discussions). This is to make sure that the issue you want hasn't already been implemented or being worked on in newer versions. Any new API or changes to existing API should be discussed to avoid inconsistencies.
+Contribution of any form is welcome! Please see the list below on how you can contribute to the project. 
 
 * Provide or suggest an implementation of an issue from [GitHub Issues](https://github.com/gluonhq/scenebuilder/issues).
 * Proof read the [public documentation](https://github.com/gluonhq/scenebuilder/wiki) for errors, ambiguities and typos.
 * Create an issue or suggest a feature backed up by a use case.
 * Add missing tests.
+* Quality-test early release builds.
+
+## Contribution Workflow
+
+1. Pick an unassigned [open issue](https://github.com/gluonhq/scenebuilder/issues).
+2. Comment on it to say you would like to contribute to fixing it and propose a plan (if appropriate).
+3. A maintainer will confirm the issue is valid and can be assigned to you for a fix.
+4. You produce a Pull Request following the Standards below.
+5. Once you tick all check boxes on the Pull Request template, it will be reviewed by a maintainer or a community member.
+6. Once the Pull Request has at least one approval, it will be ready for a merge.
```

Comment:
I think some PR's require > 1 approval.

Reply:
Yes, I will update this guidance once we define / agree #455 and therefore define the difference between `minor` and `major`, or whatever names we agree. Based on that, we can then agree that `minor` needs at least 1 approval, `major` needs at least `n`, whatever `n` is.